js/src/widgets/contextControls.js

```diff
@@ -69,11 +175,37 @@
                                    '<a class="mirador-osd-edit-mode hud-control" role="button" aria-label="Make a new annotation using mouse">',
                                    '<i class="fa fa-lg fa-edit"></i>',
                                    '</a>',
-                                   '{{/if}}',
-                                   '<a class="mirador-osd-refresh-mode hud-control" role="button" aria-label="Refresh annotations">',
+                                   '<a class="hud-control draw-tool" style="color:#abcdef;">',
+                                   '|',
+                                   '</a>',
+                                   '{{#each tools}}',
```

Comment:
We still need to check for a bad value or empty array, since the developer could pass those in for ```availableAnnotationDrawingTools``` through the configuration